### PR TITLE
User listing

### PR DESCRIPTION
**Pull request by gunjankothari**

### CODE REVIEW

File: src/admin/pages/users/usersInfoGrid.vue [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    height: 19px;
+    width: 19px;
+    left: 4px;
+    bottom: 4px;
+    background-color: white;
+    -webkit-transition: .4s;
+    transition: .4s;
+}
+
+input:checked + .slider {
```
Comment (gunjankothari) [inline src/admin/pages/users/usersInfoGrid.vue]: trial comment

File: src/admin/pages/users/users-store.js [hunk 2]
```diff
@@ -0,0 +1,185 @@
+import Vue from 'vue';
+import Vuex from 'vuex';
+Vue.use(Vuex);
+
+export default new Vuex.Store({
+
+    state: {
+        users: [
```
Comment (gunjankothari) [inline src/admin/pages/users/users-store.js]: Move this data to json file in data folder and import here.

File: src/admin/pages/users/usersInfoGrid.vue [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+        </kendo-grid>
+    </div>
+</div>
+</template>
+
+<script>
+    import usersStore from '../users/users-store';
+
+    export default {
+        name: "usersInfoGrid",
```
Comment (gunjankothari) [inline src/admin/pages/users/usersInfoGrid.vue]: this name is not mandatory here. It is optional. 
We shall keep it open ended.

File: src/admin/pages/users/usersInfoGrid.vue [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+                })
+            }
+        }
+
+
+
+    }
+</script>
+
+<style type="text/css">
```
Comment (gunjankothari) [inline src/admin/pages/users/usersInfoGrid.vue]: Use less instead of css.

File: src/admin/routes.js [hunk 5]
```diff
@@ -1,7 +1,10 @@
 import AdminPage from '/';
 import HomePage from './pages/home';
-import GroupingPage from './pages/grouping'
-import UsersPage from './pages/users';
+
+// Lazy Loading Vue Components
+function loadView (view) {
```
Comment (gunjankothari) [inline src/admin/routes.js:5]: I like it. Good approach.

File: src/layouts/sidebarRight.vue [hunk 6]
```diff
@@ -5,14 +5,14 @@
                 Header Should come here.
             </slot>
         </div>
-        <div class="right-panel-wrapper" :class="{ 'hidden': rightHidden }">
+      <!--  <div class="right-panel-wrapper" :class="{ 'hidden': rightHidden }">
```
Comment (gunjankothari) [inline src/layouts/sidebarRight.vue:8]: Uncomment this, we are anyways not going to use this file for admin.

File: src/main.js [hunk 7]
```diff
@@ -24,6 +34,7 @@ new Vue({
     el: '#app',
     router,
     store,
+    usersStore,
```
Comment (gunjankothari) [inline src/main.js]: wrong place, remove this userStore from here. 
I think you are already using usersStore in admin/store

File: src/admin/pages/users/users.vue [hunk 8]
```diff
@@ -5,16 +5,19 @@
                 <breadcrumb></breadcrumb>
             </b-col>
         </b-row>
+        <users-info-grid></users-info-grid>
```
Comment (gunjankothari) [inline src/admin/pages/users/users.vue]: Use Bootstrap Grid Row here.
```
<b-row>
       <b-col>
             <users-info-grid></users-info-grid>
        </b-col>
</b-row>
```

File: src/admin/pages/users/users-store.js (hunk 2 above)
Comment (RahulAadroja) [reply src/admin/pages/users/users-store.js]: Ok Sure

File: src/admin/pages/users/usersInfoGrid.vue (hunk 3 above)
Comment (RahulAadroja) [reply src/admin/pages/users/usersInfoGrid.vue]: Ok

File: src/admin/pages/users/usersInfoGrid.vue (hunk 4 above)
Comment (RahulAadroja) [reply src/admin/pages/users/usersInfoGrid.vue]: Ok

File: src/admin/routes.js (hunk 5 above)
Comment (RahulAadroja) [reply src/admin/routes.js:5]: Thank you

File: src/layouts/sidebarRight.vue (hunk 6 above)
Comment (RahulAadroja) [reply src/layouts/sidebarRight.vue:8]: ok

File: src/main.js (hunk 7 above)
Comment (RahulAadroja) [reply src/main.js]: yes

**Pull request closed without merging**